encrypted-media/resources/clearkey-retrieve-persistent-license.html

```diff
@@ -0,0 +1,92 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset=utf-8>
+    <meta name="timeout" content="long">
+    <title>Encrypted Media Extensions: Retrieve stored persistent-license</title>
+    <link rel="help" href="https://w3c.github.io/encrypted-media/">
+
+    <!-- Helper scripts for Encrypted Media Extensions tests  -->
+    <script src=/encrypted-media/util/utils.js></script>
+    <script src=/encrypted-media/util/fetch.js></script>
+    <script src=/encrypted-media/util/utf8.js></script>
+    <script src=/encrypted-media/util/testmediasource.js></script>
+
+    <!-- Message handler for Clear Key keysystem -->
+    <script src=/encrypted-media/util/clearkey-messagehandler.js></script>
+
+  </head>
+  <body>
+    <div id='log'></div>
+
+    <div id='video'>
+      <video id="videoelement" width="200px"></video>
+    </div>
+
+    <script>
+
+    window.addEventListener( 'message', function( event ) {
```

Comment:
It might be worth noting that this is from postMessage() and not the same as the MediaKeySession event.

(There were no responses to the [comments in this file before it was renamed](https://github.com/w3c/web-platform-tests/pull/3424/files/1703c17631dcce9be010c909f32178bf1564d4ac#diff-7a20c3da279e1bbc305b0a68f50a1b7a), so I've duplicated them here.)